Services/Meet/PsychologicalSupportPlatform.Meet.Application/Meetups/Commands/OrderMeetupByPsychologist/OrderMeetupByPsychologistCommandHandler.cs

```diff
@@ -0,0 +1,60 @@
+using MapsterMapper;
+using MediatR;
+using PsychologicalSupportPlatform.Common.Errors;
+using PsychologicalSupportPlatform.Meet.Domain.Entities;
+using PsychologicalSupportPlatform.Meet.Domain.Interfaces;
+
+namespace PsychologicalSupportPlatform.Meet.Application.Meetups.Commands.OrderMeetupByPsychologist;
+
+public class OrderMeetupByPsychologistCommandHandler: IRequestHandler<OrderMeetupByPsychologistCommand, int>
+{
+    private readonly IMeetupRepository _meetupRepository;
+    private readonly IScheduleCellRepository _scheduleCellRepository;
+    private readonly IMapper _mapper;
+    
+    public OrderMeetupByPsychologistCommandHandler(IMeetupRepository meetupRepository, IScheduleCellRepository scheduleCellRepository, IMapper mapper)
+    {
+        _meetupRepository = meetupRepository;
+        _scheduleCellRepository = scheduleCellRepository;
+        _mapper = mapper;
+    }
+
+    public async Task<int> Handle(OrderMeetupByPsychologistCommand request, CancellationToken cancellationToken)
+    {
+        var chosenScheduleCell = await _scheduleCellRepository.GetByIdAsync(request.MeetupDto.ScheduleCellId);
```

Comment:
move all possible checks to validator as we discussed

Reply:
fixed